src/components/NavigationDesktop/NavigationItemDesktop.js

```diff
@@ -81,7 +81,7 @@ class NavigationItemDesktop extends Component {
       this.setState({ isSubNavOpen: !this.state.isSubNavOpen });
     } else {
       const path = this.linkPath();
-      Router.pushRoute(path, { slug: navItem.slug });
+      Router.pushRoute(path, { slug: navItem.navigationItem.data.url });
```

Comment:
I don't think having `slug` passed in here is needed or correct since the `path` from the server should now be the full URL? Just `Router.pushRoute(path)` should work.